indexing.sql

```diff
@@ -0,0 +1,48 @@
+
+\i
+scripts/car_models.sql
+\i scripts/car_model_data.sql
+\i scripts/car_model_data.sql
+\i scripts/car_model_data.sql
+\i scripts/car_model_data.sql
+\i scripts/car_model_data.sql
+\i scripts/car_model_data.sql
+\i scripts/car_model_data.sql
+\i scripts/car_model_data.sql
+\i scripts/car_model_data.sql
+\i scripts/car_model_data.sql
+
+-- CREATE INDEX car_idx 
+-- ON car_models (make_code);
```

Comment:
So you should only make an index on make_code because your where clauses only really uses that one. You could also create a index on make_code and model_code since one of your where clauses uses it but it doesn't give you that much of a faster result. Year does not see the same benefits because for each year, there's still a lot of data, so even after sorting them by year, you'd still have to do a lot of searching.